contracts/Vault.vy

```diff
@@ -1720,7 +1730,7 @@ def report(gain: uint256, loss: uint256, _debtPayment: uint256) -> uint256:
     # Update reporting time
     self.strategies[msg.sender].lastReport = block.timestamp
     self.lastReport = block.timestamp
-    self.lockedProfit = gain  - totalFees  # profit is locked and gradually released per block
+    self.lockedProfit = self._calculateLockedProfit() + gain - totalFees
```

Comment:
```suggestion
    # Profit is locked and gradually released per block
    # NOTE: compute current locked profit and replace with sum of current and new
    self.lockedProfit = self._calculateLockedProfit() + gain - totalFees
```

Reply:
From the auditors:
"the call to `_calculateLockedProfit` appears to return an identical value to `self.lockedProfit`  as the `lockedFundsRatio` will always be 0 as `self.lastReport` is updated just prior to the function call to be `block.timestamp` so `lockedFundsRatio: uint256 = (block.timestamp - self.lastReport) * self.lockedProfitDegration` will always be 0."

Reply:
4165d877af771a7918d98630d7031705a74c1142